src/ShopArticlePreorderBundle/objects/db/TPkgShopArticlePreorder.class.php

```diff
@@ -75,6 +77,9 @@ public function SendMail($oArticle = null, $sEmail = '')
      * @param string $sArticleId
      *
      * @return bool
+     *
+     * @psalm-suppress UndefinedClass
+     * @FIXME References `TdbShopArticlePreorder` when it probably should be `TdbPkgShopArticlePreorder`
```

Comment:
The method calls `TdbShopArticlePreorder::LoadFromFields` - but `TdbShopArticlePreorder` does not exist. Should this be calling `TpkgShopArticlePreorder` which does exist?

Reply:
Yes, I guess so, but I think the fixme comment is sufficient here as this problem is not the focus of this PR